recipes/hyperscan/all/test_package/CMakeLists.txt

```diff
@@ -2,8 +2,17 @@ cmake_minimum_required(VERSION 3.1)
 project(PackageTest CXX)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+conan_basic_setup(TARGETS)
+
+find_package(hyperscan COMPONENTS hs REQUIRED)
+add_executable(hs_example hs_example.cpp)
+target_link_libraries(hs_example PRIVATE hyperscan::hs)
+set_property(TARGET hs_example PROPERTY CXX_STANDARD 11)
```

Comment:
The C++ standard should be managed via Conan.

Reply:
Removed

Reply:
I don't find example of how to specify manually the cppstd in test_package. Maybe I don't search the right thing?